text/0021-internal-data-monitoring.md

```diff
@@ -0,0 +1,182 @@
+data is from different sources - split up by each notable Creative Labs event
+(Demo Day or General Meeting) for Project 2 and split by portions of the
+main website for Project 3.
+
+Main parts of Project 1:
+* displaying important comparisons of data
+* options for user selection
+    * time period
+    * project type
+    * member major
+    * member year
+
+We could use the following data tables for each member application and time period.
+
+Member application
+```json
+{
+    "<name>": {
+        "year": "<year>",
+        "major": "<major>",
+        "email": "<email>",
+        "timestamp": "<timestamp>",
+        "project_choice_order": "[<choice_1>, <choice_2>, ...]",
+        "other_application_cycles": "[<cycle_1>, <cycle_2>, ...]"
+        ...
+    }
+}
+```
+
+Time period:
+```json
+{
+    "<quarter_id>": {
+        "number_of_projects": "<number>",
+        "number_of_members": "<number>",
+        "project_distribution": {
+            "iOS": "<number>",
+            "web app": "<number>",
+            "writing": "<number>",
+            ...
+        }
+        "grade_distribution": {
+            "first": "<number>",
+            "second": "<number>",
+            "third": "<number>",
+            "fourth": "<number>",
+            "other" : "<number>"
+        },
+        "major_distribution": {
+            "Computer Science": "<number>",
+            "Cog Sci": "<number>",
+            "Engineering": "<number>",
+            "DESMA": "<number>",
+            ...
+        },
+    }
+}
+```
+
+## What are the requirements for this project?
+
+Watchdog is a collection of internal data visualization projects. They 
+all require authentication as the data is privacy sensitive, so Firebase
+experience for authentication and data storage would be useful. The
+projects would utilize Python for data fetching and processing portions
+and use React on the frontend to display the visualization and handle
+user interactions. 
+
+Project 1 with project applications would use the [gspread API](https://docs.gspread.org/en/latest/user-guide.html)
+to obtain the data from the spreadsheets and parse through/organize the
+necessary statistics. Then, the post-processed organized data would be stored
+into Firebase, where we would use [React-Vis](https://uber.github.io/react-vis/) to display the data
+and come up with concrete conclusions and inferences.
+
+Project 2 with social media monitoring would use the [Facebook API](https://developers.facebook.com/docs/graph-api/reference/page/)
+and the [Instagram API](https://developers.facebook.com/docs/instagram-api/) to obtain the neccesary datapoints. We would
+need to map each post and advertisement run to a specific event hosted by
+Creative Labs and quantify the effect or potential correlation. 
+
+Project 3 with monitoring for the main website would require integration with
+the implementation of the site in the future.
+
+We would then be able to launch the web apps through Netlify.
```

Comment:
Honestly, I would rather you take the blurb section and bring it down here for each project! That way the description is coupled with the implementation